tests/golos.publication_rewards_tests.cpp

```diff
@@ -378,21 +420,22 @@ class reward_calcs_tester : public golos_tester {
                         }
                     }
                     for (auto& r : cur_rewards) {
-                        _state.balances[r.first].vestamount += get_converted_to_vesting(r.second);
+                        pay(_forum_name, r.first, r.second, static_cast<enum_t>(payment_t::VESTING));
```

Comment:
Why do you cast `payment_t::VESTING` to `enum_t`? You can pass it as `payment_t`.